hudi-hadoop-common/src/main/java/org/apache/hudi/common/util/ParquetUtils.java

```diff
@@ -495,7 +495,7 @@ private static BigDecimal extractDecimal(Object val, DecimalMetadata decimalMeta
    * An iterator that can apply the given function {@code func} to transform records
    * from the underneath record iterator to hoodie keys.
    */
-  private static class HoodieKeyIterator implements ClosableIterator<HoodieKey> {
+  public static class HoodieKeyIterator implements ClosableIterator<HoodieKey> {
```

Comment:
let's move it to superclass `FileFormatUtils`?

Reply:
Ok, will do.